### ISSUE TITLE
[Web] [Accessibility] [11yMAS][Screen Readers][MAS1.1.1][HighImpact][Alternative text is not provided for all Images on welcome screen.]

### ISSUE THREAD

**Issue by DasaradhaKamjula** [OfficeDev/script-lab-2017#705]
**User Experience:**
User will face difficulty if Alternative text is not provided for all images which helps user to know the purpose of the image.

**Test Environment:**
OS Build: 16299.19
Browser: Microsoft Edge 41.16299.15.0
OS Build: Mac OS Sierra 0.12.6
Browser: Safari 11.0.1,IE11
Tools Used: Developer tool
**Attachments:**
![mas1 1 1_alt is not provided](https://user-images.githubusercontent.com/33054846/32547472-75453208-c4a8-11e7-940f-f79e0e517823.png)

**Repro Steps:**
1.	Open the URL: https://script-lab.azureedge.net/.
2.	Navigate to Office dev center logo and open dev tool.
3.	Verify alternative text is provided or not for all Images in welcome screen. .

**Current Behavior:**
Alternative text is not provided for all the images in welcome screen. When user navigates in Item or scan mode screen reader is announcing it as Image.

**Expected Behavior:**
Alternative text should be provided for images like “Excel logo” or relevant name which will provide clear description about the image.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={d2d2051f-bdc8-4af7-8e18-38aae867e216}

**Suggested Fix:**
https://microsoft.sharepoint.com/teams/msenable/mas/pages/mas-details.aspx?masid=1.1.1

**Comment by KevinTCoughlin** [comment]
I don't think `alt` tags are valid on `<div>`s so it would need to be an ARIA property. But it might be better to use a group of `<button>` with `title` or anchors (tho we're not modifying state here). Keyboard accessibility would then come with that change.
